parsers/ES.py

```diff
@@ -327,10 +326,6 @@ def fetch_exchange(
     print(fetch_production("ES-CN-HI"))
 
     # Balearic Islands
-    print("fetch_consumption(ES-IB)")
```

Comment:
Why are those removed ?

Reply:
Let's keep them if anyone wants to test this code.

Reply:
We don't actually have a config for ES-IB so I removed the imported class for a small performance gain (miniscule at best but still).

So I removed these as well to not cause any confusion, this also brings it in like with how we do it for the other island group.

But I can add it back if you prefer.

Reply:
Ah indeed, I didn't saw that, agree better to remove it then.